contracts/EncodeDAOCore.sol

```diff
@@ -3,16 +3,57 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/access/AccessControl.sol";
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
 
 contract EncodeDAOCore is ERC721URIStorage, AccessControl {
+    using Counters for Counters.Counter;
+
+    /// Events
+    event ProposeIssue(
+        address indexed _from,
+        string name,
+        uint256 fundingMinimum,
```

Comment:
Would a minimum variable need the type of `uint256`? 
This could be a lower uint type😄

Reply:
Valid point 👍